src/Apps/Artwork/Components/ArtworkZendesk.tsx

```diff
@@ -33,11 +35,15 @@ const ArtworkZendesk: FC<ArtworkZendeskProps> = ({ artwork }) => {
   // Don't display on inquiry artworks
   if (isInquireable && !isAcquireable && !isOfferable) return null
 
-  if (!price || !isExceededZendeskThreshold(price.major, price.currencyCode)) {
+  if (!price || !exceedsChatSupportThreshold(price.major, price.currencyCode)) {
     return null
   }
 
-  return <ZendeskWrapper mode={isInAuction ? "auction" : "default"} />
+  return getENV("SALESFORCE_CHAT_ENABLED") ? (
+    <SalesforceWrapper />
+  ) : (
+    <ZendeskWrapper mode={isInAuction ? "auction" : "default"} />
+  )
```

Comment:
Does this mean that in Zendesk we had a specific outcome in case of auctions and in SF, this difference is not relevant anymore?

Reply:
I asked this question some time ago with no response. I'm resurfacing the question once again on the migration channel.

Reply:
oh I missed that! Thank you for bringing it up again 🙏